docs/index.md

```diff
@@ -49,6 +49,7 @@ ThisBuild / tlBaseVersion := "0.4" // your current series x.y
 
 ThisBuild / organization := "org.typelevel"
 ThisBuild / organizationName := "Typelevel"
+ThisBuild / startYear := Some(2022)
```

Comment:
I wanna make this a variable :P

Reply:
Haha, so that it's always the current year?

Reply:
Right! Anyway feel free to skip, I'll do it while I muck about with coursier 😏

Reply:
Sounds good, you have 193 days until it becomes an issue 😆